packages/generate-persisted-query-manifest/cli.js

```diff
@@ -65,7 +66,20 @@ program
     );
     writeFileSync(outputPath, JSON.stringify(manifest, null, 2));
 
-    console.log(`Manifest written to ${outputPath}`);
+    if (manifest.operations.length === 0) {
```

Comment:
I opted for a hybrid approach by logging a warning in addition to showing the number of matched operations in the log below. This gives us a chance to educate the user on where to find more information on configuring the glob pattern.

Here is what it looks like in the terminal:
<img width="1147" alt="Screenshot 2023-08-23 at 5 29 24 PM" src="https://github.com/apollographql/apollo-utils/assets/565661/f011d749-26b0-49e3-ba15-5ba565f603b4">

Feedback welcome if that warning is too distracting or too wordy.